release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -101,16 +101,47 @@ With this release, the installation program uses Cluster API instead of Terrafor
 [id="ocp-4-17-web-console_{context}"]
 === Web console
 
+[id="ocp-4-17-ocp-lightspeed_{context}"]
+==== {ols} is available in the web console
+
+Starting with {product-title} 4.16, {ols} is available for use in the web console.
+
+//https://github.com/openshift/openshift-docs/pull/81581
+//For more information, see xref:../web_console/capabilities_products-web-console.html#openshift-lightspeed-web-console_capabilities-web-console[Enhancing your experience in the {product-title} web console by using {ols-official}].
+
 [id="ocp-4-17-administrator-perspective_{context}"]
 ==== Administrator perspective
 
 This release introduces the following updates to the *Administrator* perspective of the web console:
 
+* Deprecated Operators are displayed in the OperatorHub before and after installation.
```

Comment:
```suggestion
* Deprecated Operators will be displayed along with a warning icon in the OperatorHub before and after installation.
```

Reply:
We may need to clarify what's the difference between deprecated operators and normal operators(made a suggestion above)
![Screenshot 2024-09-25 at 10 37 15 AM](https://github.com/user-attachments/assets/cba74f51-9c2f-4b2c-b408-763253da017a)